Makefile

```diff
@@ -125,12 +125,13 @@ namespace:
 deploy: certs
 	@{ \
 	set -e ;\
-	TEMP_FILE=/tmp/authorino-deploy-$$(openssl rand -hex 4).yaml ;\
+	TEMP_DIR=$(PROJECT_DIR)/tmp ;\
+	mkdir $$TEMP_DIR ;\
+	TEMP_FILE=$$TEMP_DIR/authorino-deploy.yaml ;\
 	cp $(AUTHORINO_CR) $$TEMP_FILE ;\
 	sed -i "s/\$$(AUTHORINO_INSTANCE)/$(AUTHORINO_INSTANCE)/g;s/\$$(TLS_ENABLED)/$(TLS_ENABLED)/g" $$TEMP_FILE ;\
-	$(EDITOR) $$TEMP_FILE ;\
```

Comment:
Why removing the editor step?

Reply:
Added it back after setting a defaulting to vim in case EDITOR is empty